actions/destinations/models.go

```diff
@@ -0,0 +1,16 @@
+package destinations
+
+import (
+	"github.com/bitcoin-sv/spv-wallet/engine"
+)
+
+type CreateDestination struct {
+	Metadata engine.Metadata `json:"metadata"`
+}
+
+type UpdateDestination struct {
+	Id            string          `json:"id"`
```

Comment:
Lint error:
`var-naming: struct field Id should be ID`

Reply:
Done